packages/fxa-payments-server/src/lib/AppLocalizationProvider.tsx

```diff
@@ -9,6 +9,12 @@ import { LocalizationProvider } from '@fluent/react';
 import React, { Component } from 'react';
 
 import availableLocales from 'fxa-shared/l10n/supportedLanguages.json';
```

Comment:
does anyone know why `en-US` isn't in the json file?

Reply:
@clouserw do you know?  https://github.com/mozilla/fxa-content-server-l10n/tree/master/locale/en_US/LC_MESSAGES does exist.